packages/now-static-build/test/fixtures/10a-gatsby-redirects/gatsby-config.js

```diff
@@ -1,3 +1 @@
-module.exports = {
-  plugins: ["gatsby-plugin-now"]
-};
+module.exports = {};
```

Comment:
Will this test work even if this file is removed?

Reply:
Perhaps you should modify the `10-gatsby` project to prove that one works with with redirects too?

Reply:
Yes, that's a great point 👍

Reply:
(it should work if the file is removed, i should use this test for that!)